server/src/main/java/io/crate/expression/scalar/arithmetic/RoundFunction.java

```diff
@@ -62,6 +69,36 @@ public static void register(Functions.Builder module) {
                     }
                 }
             );
+
+            module.add(
+                Signature.builder(NAME, FunctionType.SCALAR)
+                    .argumentTypes(DataTypes.DOUBLE.getTypeSignature(),
+                        DataTypes.INTEGER.getTypeSignature())
```

Comment:
I wonder if the first argument type should also be `numeric`, or if we should add a second overload to avoid precision loss if the argument is a `numeric` with higher precision.

Reply:
I've added a numeric overload and addressed cases involving long values, which could lose precision when cast to double. Could you please take another look?